media/base/decoder_buffer.cc

```diff
@@ -47,7 +72,13 @@ DecoderBuffer::DecoderBuffer(const uint8_t* data,
 }
 
 DecoderBuffer::DecoderBuffer(std::unique_ptr<uint8_t[]> data, size_t size)
+#if !BUILDFLAG(IS_COBALT)
     : data_(std::move(data)), size_(size) {}
+#else // !BUILDFLAG(IS_COBALT)
+    : size_(size) {
+      // TODO(b/374822966): revisit DecoderBufferAllocator once rebase to m126+
```

Comment:
Do we need to copy the data?

Reply:
No, as we don't use it, so we don't need to copy data here.